x-pack/plugins/observability/public/components/app/cases/callout/index.tsx

```diff
@@ -12,7 +12,7 @@ import { CallOut } from './callout';
 import { ErrorMessage } from './types';
 import { createCalloutId } from './helpers';
 import { useMessagesStorage } from '../../../../hooks/use_messages_storage';
-import { OBSERVABILITY } from '../../../../../common/const';
+import { observabilityAppId } from '../../../../../common/const';
```

Comment:
just a nit, but calling this file `const` looks off to me, can we name it `constants.ts`?

Reply:
👍🏻 I think this was added when cases was added. I might just move it to be in common/index.